### ISSUE TITLE
Have you removed the overlapping classes between GLD and Oxford/Pairs?

### ISSUE THREAD

**Issue by anguoyuan** [MCC-WH/Token#1]
Hi, authors. Impressive paper and great results! Have you removed the overlapping classes between GLDv2 and Oxford/Paris during training? In GeM and DELF papers, they claim that they removed the overlapping classes, but I did not find this claim in your paper. I recognize your contribution no matter you removed the overlapping or not. Your model outperforms DELG a lot, which is trained on the same GLDv2. I am just out of interest. Thank you!

**Comment by MCC-WH** [comment]
Thanks for your interest in our work. We don't remove overlaps from the GLDv2 dataset and the Roxf/Rpar dataset. We divide the dataset according to the settings in the DELG paper. Recently, SuperFeature points out that there are some images from the test set in GLDv2, and thus GLDv2 is not suitable as a training set. Neither GLDv2, DELG and DOLG mention that they remove the overlaps in GLDv2. I have also reproduced many methods in my paper, which use the same training set as our method. I will re-train later using GLDv2 with the overlaps removed to re-measure the performance of different models.

**Comment by anguoyuan** [comment]
Thanks for your quick response! I think your approach is well-motivated. Expect to see the performance of your model after removing overlapping classes~ ^^
